File: Funcional/ATV_1_LISTA_1/exercicio9.py
categorias = [
    "INFANTIL A",
    "INFANTIL B",
    "JUVENIL A",
    "JUVENIL B",
    "SENIOR"
]

def verificaCategoria(idade):
    if idade >= 5 and idade <=7:
        return("A categoria do competidor eh: {}".format(categorias[0]))
    elif idade >= 8 and idade <= 10:
        return("A categoria do competidor eh: {}".format(categorias[1]))
    elif idade >= 11 and idade <= 13:
        return("A categoria do competidor eh: {}".format(categorias[2]))
    elif idade >= 14 and idade <= 17:
        return("A categoria do competidor eh: {}".format(categorias[3]))
    elif idade >= 18:
        return("A categoria do competidor eh: {}".format(categorias[4]))
    return("Categoria não listada")
    
print(verificaCategoria(18))

File: Funcional/ATV_1_LISTA_1/src/exercicio9.hs
categorias = ["INFANTIL A", "INFANTIL B", "JUVENIL A", "JUVENIL B", "SENIOR"]

verificaCategoria idade
    | idade >= 5 && idade <= 7 = categorias !! 0
    | idade >= 8 && idade <= 10 = categorias !! 1
    | idade >= 11 && idade <= 13 = categorias !! 2
    | idade >= 14 && idade <= 17 = categorias !! 3
    | idade >= 18 = categorias !! 4
    | otherwise = "Categoria nao listada"

main = do
    print (verificaCategoria 5)
    print (verificaCategoria 8)
    print (verificaCategoria 11)
    print (verificaCategoria 14)
    print (verificaCategoria 18)
    print (verificaCategoria 20)